ingress/src/main/java/com/redhat/service/bridge/ingress/api/IngressAPI.java

```diff
@@ -38,7 +38,7 @@
                 scheme = "Bearer")
 })
 @SecurityRequirement(name = "bearer")
-@Path("/events")
+@Path("/")
```

Comment:
The `Ingress` network settings are now at the location of `xyz/events`.

Reply:
wdym?